extensions/wikia/Insights/models/InsightsDeadendModel.php

```diff
@@ -1,30 +1,13 @@
 <?php
 
 class InsightsDeadendModel extends InsightsQuerypageModel {
+	const INSIGHT_ID = 'deadend';
```

Comment:
I think that we should be consistent in terms of the subpages name. They used to be "subpages", then "categories" and now it's "ID". I'd say - let's pick one and stick to it in all contexts.